lib/verdict/experiment.rb

```diff
@@ -35,6 +36,10 @@ def store_unqualified?
     @store_unqualified
   end
 
+  def timestamps_out_of_band?
```

Comment:
I renamed to `manual_assignment_timestamps` to make it more clear than its the assignments, not hte experiment, that has manually set timestamps.